packages/cosmic-swingset/src/block-manager.js

```diff
@@ -6,16 +6,52 @@ import { assert, details as X } from '@agoric/assert';
 
 import * as BRIDGE_ID from '@agoric/vats/src/bridge-ids.js';
 
+import { makeStoredSubscription, makeSubscriptionKit } from '@agoric/notifier';
 import * as ActionType from './action-types.js';
 import { parseParams } from './params.js';
 
+import '@agoric/notifier/exported.js';
+
 const console = anylogger('block-manager');
 
 // Artificially create load if set.
 const END_BLOCK_SPIN_MS = process.env.END_BLOCK_SPIN_MS
   ? parseInt(process.env.END_BLOCK_SPIN_MS, 10)
   : 0;
 
+/** @typedef {Record<string, unknown>} InstallationNotification */
+
+/**
+ * @template T
+ * @param {T} initialValue
+ * @param {StorageNode} [storageNode]
+ */
+const makeStoredPublisher = (initialValue, storageNode) => {
+  let providedPublication;
```

Comment:
Is there a reason not to initialize more fully up front? Setting up the dependencies should be inexpensive... is it instead a case of chainStorage writes happening too soon (and if so, is that fixed by switching to `makeEmptyPublishKit()`, which has no initial value)?
```suggestion
  const { publication, subscription } = makeSubscriptionKit(initialValue);
  makeStoredSubscription(subscription, storageNode);
  return { publish: publication.updateState };
```

Reply:
In #6103, I made casting more resilient to uninitialized storage, but at this height of the PR stack, the `{coming: 'soon'}` value substantially greases the wheels.